charts/mimir-distributed/values.yaml

```diff
@@ -782,6 +782,36 @@ query_frontend:
 store_gateway:
   replicas: 1
 
+  rollout_operator:
+    enabled: false
+    max_unavailable: 10
+    zones:
+    - name: zone-a
+      nodeSelector:
+        topology.kubernetes.io/zone: zone-a
+      affinity: {}
+        # podAntiAffinity:
```

Comment:
I think affinity on the other hand should be predefined for all zones with a sane default (the one here)

Reply:
which will mean that it won't be possible to use this in single node mode though - the alternative is to define in small/large/capped-small/capped-large yamls

Which I already do for ingesters anyway